src/util/assert.h

```diff
@@ -57,4 +57,6 @@ inline void mixxx_release_assert(const char* assertion, const char* file, int li
 #define DEBUG_ASSERT(cond)
 #endif
 
+/// Same as DEBUG_ASSERT, but if MIXXX_DEBUG_ASSERTIONS_FATAL is disabled
+/// run the specified fallback funtion.
```

Comment:
```suggestion
/// run the specified fallback function. In most cases you should probably use this rather than DEBUG_ASSERT. Only use DEBUG_ASSERT if there is no appropriate fallback.
```